installer/dev/install.cpp

```diff
@@ -277,6 +277,23 @@ namespace WindowsAppRuntimeInstaller
             return;
         }
 
+        PackageManager packageManager;
+        const auto package{ packageManager.FindPackageForUser(L"",packageProperties->fullName.get()) };
+        if (package)
+        {
+            if (WI_IsFlagClear(options, WindowsAppRuntimeInstaller::Options::RepairPackages))
+            {
+                std::wcout << packageProperties->fullName.get() << " package is already installed. Skipping re-installing it." << std::endl;
```

Comment:
The message says "re-install" but the option is "repair". The _mechanism_ to repair is re-install but that's an implementation detail

No need for multiple separate print statements

No need to mention both short and long form options. Just mention long, and --help explains the variants (if any)

Narrow strings with wcout is usually requires it do narrow->wide conversion to output. Generally do everything with wide strings.

SUGGESTION:

```
std::wcout << packageProperties->fullName.get() << L" package is already installed." << std::endl
           << L"INFO: If you want to repair a package use the --repair option." << std::endl;
```

Or if you want to go old school

```
wprintf(L"%s package is already installed.\n"
        L"INFO: If you want to repair a package use the --repair option.\n",
        packageProperties->fullName.get());